app/src/main/java/ch/epfl/sdp/blindly/utils/CheckInternet.kt

```diff
@@ -0,0 +1,16 @@
+package ch.epfl.sdp.blindly.utils
+
+import android.content.Context
+import android.net.ConnectivityManager
+
+class CheckInternet {
+
+    companion object {
+        const val TAG = "CheckInternet"
+
+        fun internetIsConnected(context: Context): Boolean {
```

Comment:
It might be better to rename this function to "isInternetAvailable" like a boolean value